### PR TITLE
#15:style Team and Price

### PR DESCRIPTION
**Pull request by PhamNhuQuocBao**
image
image

### CODE REVIEW

File: src/stylesheets/_reset.css [hunk 1]
```diff
@@ -11,3 +32,53 @@ ul,
...
+  font-weight: 400;
+  font-size: var(--font-size-subtitle);
+  text-transform: uppercase;
+  text-align: center;
+  padding: 15px 37px;
+}
+
+.gird {
+  display: grid;
+  padding: 3.3125rem 11.375rem 5rem;
```
Comment (demons1502) [inline src/stylesheets/_reset.css:55]: ?

Comment (Lanng21) [inline src/stylesheets/_reset.css:55]: 11rem? too large

File: src/stylesheets/components/_cardPrice.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+.card-detail p {
+  padding: 0.25rem;
+}
+
+.btn-card {
+  border: none;
+}
+
+.bg-card_primary {
+  background: #ebcb80;
```
Comment (Lanng21) [inline src/stylesheets/components/_cardPrice.css:35]: color here

File: src/stylesheets/components/_secPrice.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+.section-card {
+  background-color: var(--bg-color);
+}
+
+.price-title {
+  padding-top: 4.25rem;
+}
+
+.price-title .title {
+  color: #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/components/_secPrice.css:12]: same

File: src/stylesheets/layouts/_header.css [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+
+.header_top-navigation {
+  display: flex;
+  align-items: center;
+}
+
+.header_top-navigation a {
+  text-transform: uppercase;
+  font-size: 18px;
+  color: #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+ul .item {
+  margin: 0 22px;
+  text-align: center;
+}
+
+.item:last-child {
+  padding: 15px 25px;
+  border: 2px solid #ffffff;
+  border-radius: 50px;
+  width: 180px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: check width

File: src/stylesheets/layouts/_header.css [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: var(--text-color-primary);
+  opacity: 0.3;
+}
+
+.header_bottom-main {
+  height: 768px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: height 768 ?

File: src/stylesheets/layouts/_header.css [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+form {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 60px;
+}
+
+.input-email {
+  width: 270px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+  color: var(--text-color-binary);
+}
+
+.icon {
+  display: flex;
+  justify-content: center;
+}
+
+.icon div {
+  width: 80px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: Don't fix width and height in pixels

File: src/stylesheets/layouts/_header.css [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+  ul .item {
+    margin: 0 15px;
+  }
+
+  .header_top-navigation a {
+    font-size: 11px;
+  }
+
+  .item:last-child {
+    /* height: 40px; */
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: remove

File: src/stylesheets/layouts/_header.css [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+    width: 148px;
+    padding: 11px;
+    margin-left: 0;
+  }
+
+  .title {
+    font-size: 32px;
+    width: 405px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 11]
```diff
@@ -0,0 +1,270 @@
...
+  }
+
+  .subtitle {
+    font-size: 11px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .input-email {
+    width: 228px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+  .hamburger-menu {
+    display: block;
+    display: flex;
+    align-items: center;
+    margin-right: 20px;
+  }
+
+  .title {
+    font-size: 20px;
+    width: 190px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 13]
```diff
@@ -0,0 +1,270 @@
...
+    font-weight: 400;
+  }
+
+  .subtitle {
+    font-size: 11px;
+  }
+
+  form {
+    display: block;
+    width: 190px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

**Pull request merged**